### PR TITLE
Larry/awstoazure

### PR DESCRIPTION
**Pull request by lbrader**
Updated table for 1st quarter 2017

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/aws-professional/services.md](https://github.com/mspnp/architecture-center/blob/larry/awstoazure/docs/aws-professional/services.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C17%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201703171735375687-8%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/aws-professional/services.md [hunk 1]
```diff
@@ -19,142 +19,210 @@ ms.date: 12/29/2016
...
+| **&nbsp;**                      | Lambda \@ Edge                  | None                                                                                                                                                                                                                                                                                                                                                                         | Runs Lambda functions on AWS Edge locations in response to CloudFront events.                                                                                                                                                                   |
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: Should this be Lambda@Edge?

Comment (lbrader) [inline docs/aws-professional/services.md]: yes

File: docs/aws-professional/services.md [hunk 2]
```diff
@@ -19,142 +19,210 @@ ms.date: 12/29/2016
...
+| Hybrid storage                     | Storage Gateway                                           | [StorSimple](https://azure.microsoft.com/en-us/services/storsimple/)                                                                                                                                                                                                                                                         | Integrates on-premises IT environments with cloud storage. Automates data management and storage, plus supports in disaster recovery.                                                                                                                                                               |
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: should this say "and supports disaster recovery" ?

File: docs/aws-professional/services.md [hunk 3]
```diff
@@ -19,142 +19,210 @@ ms.date: 12/29/2016
...
-|Search|Elasticsearch Service|[Search](https://azure.microsoft.com/services/search/)|Delivers full-text search and related search analytics and capabilities.|
-|Data discovery|**&nbsp;**|[Data Catalog](https://azure.microsoft.com/services/data-catalog/) |Provides the ability to better register, enrich, discover, understand, and consume data sources.|
 
 ## Mobile services
 
-|Subcategory|AWS Service|Azure Service|Description|
-|--- |--- |--- |--- |
-|Pro app development|Mobile Hub (Beta) <br/>Cognito|[Mobile Apps](https://azure.microsoft.com/services/app-service/mobile/)|Backend mobile services for rapid development of mobile solutions, plus provide identity management, data synchronization, and storage and notifications across devices.|
-|High-level app development|**&nbsp;**|[PowerApps](https://powerapps.microsoft.com/)|Model-driven application development for business applications with SaaS integration.|
-|Analytics|Mobile Analytics|[Mobile Engagement](https://azure.microsoft.com/services/mobile-engagement/)|Provides real-time analytics from mobile apps data, highlights app users’ behav
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: s/b "scaling web applications"

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/aws-professional/services.md](https://github.com/mspnp/architecture-center/blob/larry/awstoazure/docs/aws-professional/services.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C17%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201703171817209656-8%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/aws-professional/services.md](https://github.com/mspnp/architecture-center/blob/larry/awstoazure/docs/aws-professional/services.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C17%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201703171821521178-8%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/aws-professional/services.md](https://github.com/mspnp/architecture-center/blob/larry/awstoazure/docs/aws-professional/services.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C17%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201703171834380494-8%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**